### PR TITLE
add: node js and next js 14 resource added

### PR DESCRIPTION
**Pull request by iamtonmoy0**
## What does this PR do?
added node js and next 14 learning resources
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description
Rabbil Hasan ,Learn with sumit and Hablu Programmer is  offering new tutorial for free which i value able for new learners
### Why is this valuable (or not)?
It is valuable to learn new trending tech in native language

### How do we know it's really free?
Yes it is free of cost
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
Please start with a fresh branch and don't change the formatting of the entire index

File: courses/free-courses-bn.md [hunk 1]
```diff
@@ -1,43 +1,44 @@
...
+  - [Laravel](#laravel)
+- [Programming paradigms](#programming-paradigms)
+- [Python](#python)
+  - [Django](#django)
+  - [Flask](#flask)
+- [Scratch](#scratch)
+- [Shell scripting](#shell-scripting)
+- [Swift](#swift)
+- [TypeScript](#typescript)
+- [WordPress](#wordpress)
```
Comment (3R1Dev) [inline courses/free-courses-bn.md:41]: You need to start fresh and don't edit this. This is already formatted as it is intended to be. If you are going to add an index or subindex it needs to follow the already established format.

If you keep changing this then the pull request will be rejected.

File: courses/free-courses-bn.md [hunk 2]
```diff
@@ -210,6 +211,8 @@
 * [Next Js Bangla Tutorial Series](https://www.youtube.com/playlist?list=PLwMeE9AWeV59vbQSIArd0-sNB9FPxlWSp) - dSkill
 * [next js bangla(বাংলা) tutorial](https://www.youtube.com/playlist?list=PLQvUYGXiwrskS_C3MOeW0rOVB5Ny2MCR2) - Faazle Rabbi
 * [Next js tutorial in bangla](https://www.youtube.com/playlist?list=PLkmCJMhveta1PihBgW4MpYxFlyzdSjlNC) - Sabeek Bin Sayeed
+* [Next JS  14 Full Course in bangla](https://www.youtube.com/watch?v=XnIm8mb7WOw&list=PLNMnAEqLBwmqt3wGvc_JMqFobBZBcpv8c) - Hablu Programmer
```
Comment (3R1Dev) [inline courses/free-courses-bn.md:214]: This needs to be alphabetical order, based on the tittle you might need to add it in the beginning and not the end.

File: courses/free-courses-bn.md [hunk 3]
```diff
@@ -220,6 +223,7 @@
 * [Node.js Tutorial Bangla  Series for Beginners](https://youtube.com/playlist?list=PLHiZ4m8vCp9PHnOIT7gd30PCBoYCpGoQM) - Learn With Sumit
 * [NodeJS Tutorial Online](https://www.youtube.com/playlist?list=PLEYpvDF6qy8ZHMhSqsdo_Tge0CDxxXd1w) - Foyzul Karim
 * [Raw Node JS Project in Bangla ( বাংলা ) - Uptime Monitoring API](https://youtube.com/playlist?list=PLHiZ4m8vCp9OmVWU2Qf9tZgKdyzoubOpj) - Learn With Sumit
+* [Node.js Tutorial Bangla | Express.js Tutorial Bangla](https://www.youtube.com/playlist?list=PLkyGuIcLcmx2qXaZkjCL8-P78i2J5rDOa)-Rabbil Hasan
```
Comment (3R1Dev) [inline courses/free-courses-bn.md:226]: This also needs to be in alphabetical order, try swapping this with line 225.

**Review by 3R1Dev — CHANGES_REQUESTED** [review]
Please review the comments and make the appropriate changes.

**Pull request closed without merging**